src/components/structures/RoomView.js

```diff
@@ -717,6 +716,7 @@ module.exports = React.createClass({
         // refresh the conf call notification state
         this._updateConfCallNotification();
         this._updateDMState();
+        this._checkIfAlone(this.state.room);
     }, 500),
 
     _checkIfAlone: function(room) {
```

Comment:
Actually, this should now be implemented with roomstate getJoinedMemberCount + getInvitedMemberCount, which should be more performant and also more correct in case of lazy loading.

Reply:
heh, good point